### ISSUE TITLE
Ends of songs are cut off

### ISSUE THREAD

**Issue by julianfairfax** [vixalien/muzika#58]
It seems when a song ends, and the next in the queue plays, the end of the first song is cut off. This is probably best demonstrated by testing yourself.

**Comment by vixalien** [comment]
yes. thanks for reporting this! I have already noticed this and it appears to be some kind of issue with the MPEG-DASH manifest we're generating. It can be (sometimes) easily solved by changing audio quality to something other than `auto`.

I'll be working for an in-depth fix ASAP

**Comment by vixalien** [comment]
as a workaround for this annoying bug, please try updating the app and change audio quality to something different from `auto`.

**Comment by onny** [comment]
Also experience this on OnePlus 6 postmarketOS Linux phone 👀

**Comment by vixalien** [comment]
Let me reopen this bug. It is more related to Gstreamer. I will need to see if version 1.24 fixes that.
